contrib/pax_storage/src/cpp/access/pax_access_handle.cc

```diff
@@ -0,0 +1,1217 @@
+                                    GetActiveSnapshot(), 0, nullptr);
+          while (HeapTupleIsValid(tuple = systable_getnext(scan))) {
+            Datum datum;
+            bool isnull;
+
+            datum = heap_getattr(tuple, Anum_pg_class_relam, pax_rel->rd_att,
+                                 &isnull);
+            if (isnull || DatumGetObjectId(datum) != PAX_TABLE_AM_OID) continue;
+
+            datum = heap_getattr(tuple, Anum_pg_class_oid, pax_rel->rd_att,
+                                 &isnull);
+            Assert(!isnull);
+            Oid relid = DatumGetObjectId(datum);
+            relids = lappend_oid(relids, relid);
+          }
+          systable_endscan(scan);
+          table_close(pax_rel, AccessShareLock);
+#else
+          auto pax_aux_rel =
+              table_open(PAX_TABLES_RELATION_ID, AccessShareLock);
+          scan = systable_beginscan(pax_aux_rel, InvalidOid, false,
+                                    GetActiveSnapshot(), 0, NULL);
+
+          while ((tuple = systable_getnext(scan)) != NULL) {
+            Datum datum;
+            bool isnull;
+            datum = heap_getattr(tuple, ANUM_PG_PAX_TABLES_RELID,
+                                 pax_aux_rel->rd_att, &isnull);
+            Assert(!isnull);
+            Oid relid = DatumGetObjectId(datum);
+            relids = lappend_oid(relids, relid);
+          }
+          systable_endscan(scan);
+          table_close(pax_aux_rel, AccessShareLock);
+#endif
+
+          ListCell *lc = NULL;
+          foreach (lc, relids) {
+            Oid pax_rel_id = Oid(lfirst_oid(lc));
+
+            Relation rel = table_open(pax_rel_id, RowExclusiveLock);
+            if (relation_has_cluster_columns_options(rel)) {
+              stmt->relation = makeNode(RangeVar);
+              stmt->relation->schemaname =
+                  get_namespace_name(rel->rd_rel->relnamespace);
+              stmt->relation->relname = pstrdup(rel->rd_rel->relname.data);
+              CdbDispatchUtilityStatement((Node *)stmt,
+                                          DF_CANCEL_ON_ERROR | DF_WITH_SNAPSHOT,
+                                          GetAssignedOidsForDispatch(), NULL);
+              pfree(stmt->relation);
+              stmt->relation = NULL;
+            }
+            table_close(rel, RowExclusiveLock);
+          }
+
+          list_free(relids);
+        }
+      }
+      break;
+    }
+    default:
+      if (Gp_role == GP_ROLE_DISPATCH)
+        checkUnsupportDfsTableSpaceStmt(pstmt->utilityStmt);
+      break;
+  }
+
+  if (prev_ProcessUtilit_hook)
+    prev_ProcessUtilit_hook(pstmt, queryString, readOnlyTree, context, params,
+                            queryEnv, dest, completionTag);
+  else
+    standard_ProcessUtility(pstmt, queryString, readOnlyTree, context, params,
+                            queryEnv, dest, completionTag);
+}
+
+static void PaxCheckMinMaxColumns(Relation rel, const char *minmax_columns) {
```

Comment:
how to config all columns use with ?
for example : create table t_index_cluster(c1 int, c2 int) with (minmax_columns='all') ?

Reply:
PAX not support `minmax_columns='all'.